brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Added Starter Template to the Advanced Columns.

### Description
Trello Task - https://trello.com/c/furCIDVp/67-advanced-column-start-template-feature

### Screenshots
https://share.getcloudapp.com/E0uzP8oX

### Types of changes
New feature (non-breaking change which adds functionality)

### How has this been tested?
 - Check the number of columns as per the template.
 - Check backward compatibility. 

### Checklist:
